app/views/users/edit_info/email.html.slim

```diff
@@ -0,0 +1,9 @@
+- title t('titles.edit_info.email')
+
+
```

Comment:
Whats with the double newline here?  I notices you've been using it elsewhere too.